docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_custom-tags.adoc

```diff
@@ -0,0 +1,87 @@
+:_content-type: PROCEDURE
+:troubleshooting_builds:
+
+[id="custom-tags_{context}"]
+= Using custom tags
+
+Konflux allows tagging built images with custom tags. This tagging is handled by separate *apply-tags* task. The custom tags can be configured in two ways:
+
+. Using *konflux.additional-tags* label in your Dockerfile
```

Comment:
Would be nice to have anchor link to the sections below

Reply:
updated